packages/figma/src/codegen/targets/react/component/handlers/tabs.ts

```
@@ -1,131 +1,14 @@
 
-const { createLocalSnippetElement } = createLocalSnippetHelper("tabs");
+const { createLocalSnippetElement: createTabsLocalSnippetElement } =
+  createLocalSnippetHelper("tabs");
 
-const TABS_HUG_ITEM_KEY = "c242492543b327ceb84fa9933841512fc62a898c";
-const createTabsHugItemHandler = (_ctx: ComponentHandlerDeps) =>
-  defineComponentHandler<TabsHugItemProperties>(
-    TABS_HUG_ITEM_KEY,
-    ({ componentProperties: props }) => {
-      const states = props.State.value.split("-");
+const TABS_UNDERLINE_HUG_ITEM_KEY = "946cc52671ac3f46e20a8ff12fcc1c7f618da675";
```

Biome (2.1.2)

[error] This variable TABS_UNDERLINE_HUG_ITEM_KEY is unused.

Unused variables are often the result of an incomplete refactoring, typos, or other sources of bugs.
Unsafe fix: If this is intentional, prepend TABS_UNDERLINE_HUG_ITEM_KEY with an underscore.


(lint/correctness/noUnusedVariables)